packages/client/src/__tests__/integration/happy/exhaustive-schema/__snapshots__/test.ts.snap

```diff
@@ -1122,7 +1122,7 @@ export namespace Prisma {
   ? False
   : T extends Date
   ? False
-  : T extends Buffer
+  : T extends Uint8Array
```

Comment:
Does `TS 4.8.0` remove typing support for `Buffer` in favor of `Uint8Array`?

Reply:
Buffer is not included in the types (no type reference to node types), so it will be there only if you use node. However, since we are planning to also support Deno, I thought it would be better to check `Uint8Array` in `IsObject` instead.